tools/extra/mmlink/remote_dbg/streaming/st_dbg_ip_driver.c

```diff
@@ -245,11 +245,12 @@ int get_mgmt_rsp_data(MGMT_PACKET_HEADER *header, unsigned char **payload) {
 }
 
 void set_loopback_mode(int val) {
-    unsigned long rd = (unsigned long)IORD_64DIRECT(g_std_dbg_ip_info.ST_DBG_IP_CSR_BASE_ADDR, ST_DBG_IP_CONFIG_RESET_AND_LOOPBACK);
+
+    uint32_t rd = (uint32_t)IORD_32DIRECT(g_std_dbg_ip_info.ST_DBG_IP_CSR_BASE_ADDR, ST_DBG_IP_CONFIG_RESET_AND_LOOPBACK);
```

Comment:
How about some comments to explain what's going on?

Reply:
fixed in remote-debug-for-intel-fpga
Adjust mmio rw 64 to 32:

https://github.com/altera-opensource/remote-debug-for-intel-fpga/commit/e04a93e100e34c2eeca5aa00d60d55413970f3ca